### ISSUE TITLE
edd_render_receipt_in_browser() outputs invalid markup 

### ISSUE THREAD

**Issue by DevinWalker** [easydigitaldownloads/Easy-Digital-Downloads#3922]
Hey Guys,

Just noticed that the HTML output from the `edd_render_receipt_in_browser()` function here: https://github.com/easydigitaldownloads/Easy-Digital-Downloads/blob/master/includes/emails/template.php#L239 is not valid.

1. Title tag should be wrapped in the head per W3 standards: "Every HTML document must have a TITLE element in the HEAD section." http://www.w3.org/TR/html4/struct/global.html#edef-TITLE

2. `wp_head()` is used without a 

3. the HTML tag incorrectly closes before the body tag begins - "After document type declaration, the remainder of an HTML document is contained by the HTML element." http://www.w3.org/TR/html4/struct/global.html#edef-HTML

![10-15-2015 4-38-49 pm](https://cloud.githubusercontent.com/assets/1571635/10530357/fdf92a1e-735b-11e5-8113-3494d2c67bf1.png)

**Comment by DevinWalker** [comment]
Also I recommend adding no-cache headers like this:

```
<meta charset="utf-8" />
<meta http-equiv="cache-control" content="max-age=0" />
<meta http-equiv="cache-control" content="no-cache" />
<meta http-equiv="expires" content="0" />
<meta http-equiv="expires" content="Tue, 23 Oct 1977 05:00:00 PST" />
<meta http-equiv="pragma" content="no-cache" />
```

or

```
<?php
	//Disallows caching of the page
	header("Last-Modified: " . gmdate("D, d M Y H:i:s") . " GMT");
	header("Cache-Control: no-store, no-cache, must-revalidate"); // HTTP/1.1
	header("Cache-Control: post-check=0, pre-check=0", false);
	header("Pragma: no-cache"); // HTTP/1.0
	header("Expires: Sat, 23 Oct 1977 05:00:00 PST"); // Date in the past
	?>
```

or both... couldn't hurt. The issue is we've seen this page being cached by various plugins and hosting environments which confuses users and annoys admins when users complain to them they can't see the receipt.

LMK what you think about that...

**Comment by pippinsplugins** [comment]
Doh! Thanks for noticing and the PR! Milestoned.

**Comment by pippinsplugins** [comment]
Everything looks good here.

**Comment by pippinsplugins** [comment]
Wait no. @DevinWalker Looks like PR didn't get resubmitted to `release/2.5`. Able to do that?

**Comment by DevinWalker** [comment]
@pippinsplugins just resubmitted PR here: https://github.com/easydigitaldownloads/Easy-Digital-Downloads/pull/3951

**Comment by DevinWalker** [comment]
Add back that fancy schmancy has pr tag ;)

**Comment by pippinsplugins** [comment]
:+1:

**Comment by pippinsplugins** [comment]
Fix merged into `release/2.5`. Just needs final testing but I'm pretty sure this is ready to be closed.

**Comment by cklosowski** [comment]
Yeah this is much better. Looked it over and seems to be working fine for me. Closing.
